BrightcoveExoPlayer-Samples/ExoPlayerSampleApp/src/main/java/com/brightcove/player/samples/exoplayer/basic/MainActivity.java

```diff
@@ -3,9 +3,11 @@
 import android.os.Bundle;
 import android.util.Log;
 
+import com.brightcove.player.media.Catalog;
+import com.brightcove.player.media.VideoListener;
 import com.brightcove.player.model.Video;
+import com.brightcove.player.view.BrightcoveExoVideoView;
```

Comment:
Are there any implications in terms of copyright, fair use, etc. in naming this class like this?
That is, does "BrightcoveExoVideoView" provide a clear enough attribution to the ExoPlayer framework?
Update:
In light of the "BrightcoveExoPlayer" references elsewhere, we should consider naming this class to BrightcoveExoPlayerVideoView for maximum clarity.